src/js/playlist/loader.js

```diff
@@ -53,7 +53,7 @@ const PlaylistLoader = function() {
                         throw Error('Playlist is not an array');
                     }
                 } catch (e) {
-                    throw new PlayerError('Not a valid RSS/JSON feed', 621, e);
+                    throw new PlayerError(MSG_CANT_LOAD_PLAYER, 621, e);
```

Comment:
I think `cantPlayVideo ` would be a better message because the problem is the media files